### DISCUSSION TITLE
Browser.sh not launching Zen Browser

### DISCUSSION THREAD

**Discussion by tko76** [mylinuxforwork/dotfiles#183] [Q&A] [answered]
Hi Hyprland experts, 

I am running into an issue where I want to launch Zen Browser which I installed via Flatpak. When I hit the shortcut key of SUPER + B, which calls the ~/dotfiles/.settings/browser.sh shell script file, it does not launch Zen. My browser.sh is configured with the following command:

❯ cat ~/dotfiles/.settings/browser.sh
flatpak run io.github.zen_browser.zen

~
❯

When I enter that flatpak run command on the normal command line, Zen launches successfully. Any ideas why it is not launching via the browser.sh file or even from Waybar when I click on the icon? Thanks in advance!

**Comment by mylinuxforwork** [comment]
What happens when you execute the browser.sh directly? Cd into the folder and run ./browser.sh

**Reply by tko76** [reply, marked as answer]
It successfully launches.

**Reply by mylinuxforwork** [reply]
Can you open the settings app, system tab and check that the launch command is visible in the default browser section?

**Reply by tko76** [reply]
That fixed the shortcut command. SUPER + B now works and launches Zen. The browser setting was listed as Firefox there, and I changed it to the flatpak run command. So my settings did not match my browser.sh file. 

I also had files out of sync between the linked directories of ~/.config/ml4w/settings and ~/dotfiles/,config/ml4w which prevented the waybar quicklinks from working properly. It is now all working, thank you for the tips!
